rdflib/term.py

```diff
@@ -21,6 +21,7 @@
 
 """
 import re
+import typing
```

Comment:
If the file already uses `from typing import X` I would say it is best to do the same for `Any`

Reply:
Agreed, I'd missed the symbol import below.  Reverted.